istio/metadata.csv

```diff
@@ -196,3 +196,49 @@ istio.galley.validation.cert_key_updates,gauge,,,,Galley validation webhook cert
 istio.galley.validation.config_load,gauge,,,,k8s webhook configuration (re)loads.,0,istio,
 istio.galley.validation.config_update,gauge,,,,k8s webhook configuration updates.,0,istio,
 istio.galley.validation.passed,gauge,,,,Resource is valid.,0,istio,
+istio.citadel.secret_controller.csr_err_count,count,,,,The number of errors occurred when creating the CSR.,-1,istio,
+istio.citadel.secret_controller.secret_deleted_cert_count,count,,,,The number of certificates recreated due to secret deletion (service account still exists).,0,istio,
+istio.citadel.secret_controller.svc_acc_created_cert_count,count,,,,The number of certificates created due to service account creation.,0,istio,
+istio.citadel.secret_controller.svc_acc_deleted_cert_count,count,,,,The number of certificates deleted due to service account deletion.,0,istio,
+istio.citadel.server.authentication_failure_count,count,,,,The number of authentication failures.,-1,istio,
+istio.citadel.server.citadel_root_cert_expiry_timestamp,gauge,,,,"The unix timestamp, in seconds, when Citadel root cert will expire. We set it to negative in case of internal error.",0,istio,
+istio.citadel.server.csr_count,count,,,,The number of CSRs recerived by Citadel server.,0,istio,
+istio.citadel.server.csr_parsing_err_count,count,,,,The number of erorrs occurred when parsing the CSR.,-1,istio,
+istio.citadel.server.id_extraction_err_count,count,,,,The number of errors occurred when extracting the ID from CSR.,-1,istio,
+istio.citadel.server.success_cert_issuance_count,count,,,,The number of certificates issuances that have succeeded.,1,istio,
+istio.citadel.go.gc_duration_seconds.count,gauge,,second,,Count of the GC invocation durations.,0,istio,
+istio.citadel.go.gc_duration_seconds.quantile,gauge,,second,,Quantile of the GC invocation durations.,0,istio,
+istio.citadel.go.gc_duration_seconds.sum,gauge,,second,,Sum of the GC invocation durations.,0,istio,
+istio.citadel.go.goroutines,gauge,,thread,,Number of goroutines that currently exist.,0,istio,
+istio.citadel.go.info,gauge,,,,Information about the Go environment.,0,istio,
```

Comment:
`Information about the Go environment` seems not very clear what this metric is about.

Reply:
Yeah it's rather generic, but people using go know what's it's about I think.